apps/design/frontend/src/ballot_viewer.tsx

```diff
@@ -54,6 +56,19 @@ function SvgAnyElement({ element }: { element: AnyElement }) {
   }
 }
 
+interface DebugFlags {
+  // eslint-disable-next-line react/no-unused-prop-types
+  showGridLines: boolean;
+  // eslint-disable-next-line react/no-unused-prop-types
```

Comment:
You could get rid of these if you change the parameter type of `WriteInOptionBoxes` to `Omit<DebugInfo, keyof DebugFlags> & { width: number; height: number }`.

Reply:
01cc133ba